vhost-device-console/README.md

```diff
@@ -119,9 +136,26 @@ host# qemu-system
     ...
 ```
 
+#### Test the device with UML
+Start the daemon as above section, then run the following cmdline for
+Kernel Mode Linux [`virtio console`](https://github.com/torvalds/linux/blob/848e076317446f9c663771ddec142d7c2eb4cb43/include/uapi/linux/virtio_ids.h#L34):
+```text
+host# linux root=/dev/ubda1 rw ubd0=$YOUR-PATH/kata-ubuntu-latest.image   \
+    <normal UML options>                                             \
+    virtio_uml.device=/tmp/console.sock0:3 console=tty0 console=hvc0 \
+    init=/bin/systemd                                                \
+    systemd.unit=kata-containers.target agent.debug_console
```

Comment:
I think testing with UML does not need to depend on Kata Containers:

- Remove staff regarding kata-containers: `systemd.unit`, image, `debug_console`.
- Use `/bin/bash` instead.

```shell
$ sudo linux ubd0=<rootfs> \
    init=/bin/bash \
    console=hvc0 \
    virtio_uml.device=/tmp/console.sock0:3
```

Reply:
BTW, Kata's rootfs image can still be used here.